client/service/src/chain_ops/build_spec.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
I'm not sure if I should move the contents of this file or not.

Reply:
I think it is fine for now